packages/shared/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./stats";
```

Comment:
is there a way to namespace these exports so that they're not all imported directly from `shared` (but rather `shared/stats`)?